context("stop")

test_that("call_stop() works", {
#  expect_error(call_stop(), "message", fixed = TRUE)
})
